__tests__/storage/storage.test.js

```diff
@@ -48,7 +48,13 @@ describe('handleStop', () => {
    * @param {IStore} store
    */
   async function handleStopTest(store) {
+    async function toArray(gen) {
+      const out = [];
+      for await (const it of gen) out.push(it);
+      return out;
+    }
```

Comment:
This function is repeating itself across the test files. Should it be extracted to a helper or utils module?